src/models/m_category.erl

```diff
@@ -679,9 +676,7 @@ ensure_hierarchy(Context) ->
                     lager:warning("[~p] Ensure category found ~p new categories.", [z_context:site(Context), N]),
```

Comment:
Line 676 is too long:                     lager:warning("[~p] Ensure category found ~p new categories.", [z_context:site(Context), N]),.